src/common/operator/cast_operators.cpp

```diff
@@ -1293,6 +1294,28 @@ bool TryCastToBlob::Operation(string_t input, string_t &result, Vector &result_v
 	return true;
 }
 
+//===--------------------------------------------------------------------===//
+// Cast From UUID
+//===--------------------------------------------------------------------===//
+template <>
+string_t CastFromUUID::Operation(hugeint_t input, Vector &vector) {
+	auto uuid = Hugeint::ToUUIDString(input);
```

Comment:
Since we always know what the length of a UUID string is going to be (20 characters) first constructing a `std::string` and then copying it into the string vector is quite inefficient. It would be much more efficient to first allocate the space using `StringVector::EmptyString` and then constructing the string in-place.

I'm not sure how much we care about performance here (probably not that much), but it is something to consider as that is what we do in other cases.